<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_park</class>
 <widget class="QMainWindow" name="main_park">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LG 마곡 주차 자동 신청 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="qt_calendar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="selectedDate">
     <date>
      <year>2020</year>
      <month>2</month>
      <day>6</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>1753</year>
      <month>9</month>
      <day>14</day>
     </date>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>160</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="qt_id">
       <property name="placeholderText">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_pw">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_visitor">
       <property name="placeholderText">
        <string>방문자</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="qt_building_combo">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_phone">
       <property name="placeholderText">
        <string>연락처(xxx-xxxx-xxxx)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_company">
       <property name="placeholderText">
        <string>소속기관</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_vehicle">
       <property name="placeholderText">
        <string>차량번호(xx가xxxx)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qt_reason">
       <property name="placeholderText">
        <string>사유</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="info_save">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>주차정보저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_park">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 예약</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qt_log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
